.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright The ORAS Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report
+labels: [bug]
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: |
+        Thank you for reporting bugs to ORAS!
+  - type: textarea
+    id: environment
+    validations:
+      required: true
+    attributes:
+      label: What happened in your environment? 
+      description: "Please also attach logs here using `--debug` flag."
+  - type: textarea
+    id: expect
+    attributes:
+      label: "What did you expect to happen?"
+  - type: textarea
+    id: reproduce
+    validations:
+      required: true
+    attributes:
+      label: "How can we reproduce it?"
+      description: "Please tell us your environment information as minimally and precisely as possible."
+  - type: input
```

Comment:
Can we make this text area? Since `oras version` contains 4 lines and usually first and we usually need to look at 1st 3rd and 4th line for build information.

Reply:
Good catch. Updated.